packages/vite/src/node/cli.ts

```diff
@@ -204,22 +205,28 @@ cli
 
       const modeString =
         options.mode && options.mode !== 'development'
-          ? `  ${colors.bgGreen(` ${colors.bold(options.mode)} `)}`
+          ? `  ${styleText('bgGreen', ` ${styleText('bold', options.mode)} `)}`
           : ''
       const viteStartTime = global.__vite_start_time ?? false
       const startupDurationString = viteStartTime
-        ? colors.dim(
-            `ready in ${colors.reset(
-              colors.bold(Math.ceil(performance.now() - viteStartTime)),
+        ? styleText(
+            'dim',
+            `ready in ${styleText(
+              'reset',
+              styleText(
+                'bold',
+                String(Math.ceil(performance.now() - viteStartTime)),
+              ),
             )} ms`,
```

Comment:
worth noting that `" ms"` won't be dim here

`styleText` doesn't re-emit sequences